autogpt_platform/backend/backend/integrations/webhooks/manager_registry_test.py

```
@@ -0,0 +1,116 @@
+Python only refuses at *instantiation*, and nothing instantiates a webhook
+manager until a user actually sets up a trigger. The failure therefore lands on
+the user as::
+
+    Can't instantiate abstract class <X>WebhooksManager without an
+    implementation for abstract method 'validate_payload'
+
+These tests close that gap by exercising, for every registered provider, the
+same call the runtime makes.
+"""
+
+from typing import Type
```

Ruff (0.16.1)

[warning] `typing.Type` is deprecated, use `type` instead

(UP035)